src/sampler.cc

```diff
@@ -76,6 +76,7 @@ void Sampler::Init(uint64_t seed) {
 }
 
 #define MAX_SSIZE (static_cast<ssize_t>(static_cast<size_t>(static_cast<ssize_t>(-1)) >> 1))
+const double val_log2m26 = log(2.0) * 26;
```

Comment:
I'm wondering that when use log(2.0) * 26, the val_log2m26 is 18.021826694558577, but use Calculator the result is 7.8267798872635110755572112628368. Why does the result have such a big gap?